src/dialects/db2/query-generator.js

```diff
@@ -9,7 +10,8 @@ const { AbstractQueryGenerator } = require('../abstract/query-generator');
 const randomBytes = require('crypto').randomBytes;
 const { Op } = require('../../operators');
 
-/* istanbul ignore next */
```

Comment:
We keep this in other dialects (or at least in mssql), maybe we can keep it here as well?

Reply:
Done